integration/helpers.go

```diff
@@ -712,6 +712,49 @@ func (i *TeleInstance) startNode(tconf *service.Config, reverseTunnel bool) (*se
 	return process, nil
 }
 
+func (i *TeleInstance) StartApp(conf *service.Config) (*service.TeleportProcess, error) {
+	dataDir, err := ioutil.TempDir("", "cluster-"+i.Secrets.SiteName)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	conf.DataDir = dataDir
+	conf.AuthServers = []utils.NetAddr{
+		utils.NetAddr{
```

Comment:
nit: redundant item type, the following will work (and won't be flagged by lint):
```go
conf.AuthServers = []utils.NetAddr{
	{
		AddrNetwork: "tcp",
		Addr:        net.JoinHostPort(Loopback, i.GetPortWeb()),
	},
}
```